vernac/vernacinterp.mli

```diff
@@ -9,12 +9,25 @@
 (** Interpretation of extended vernac phrases. *)
 
 type deprecation = bool
+
 type vernac_command = Genarg.raw_generic_argument list -> unit -> unit
```

Comment:
I would attach this to `vernac_command`: This is the type of interpretation functions for `VERNAC EXTEND` blocks. It corresponds to the right-hand side of such a block and it is parametrized by the arguments of the block.